### PR TITLE
Starting to list utilities in user_guide.md

### LINKED ISSUE #213: incorporate UFS_UTILS/sorc/global_chgres.fd/USERS_GUIDE into the doxygen docs
incorporate UFS_UTILS/sorc/global_chgres.fd/USERS_GUIDE into the doxygen docs.

This is part of #191

### LINKED ISSUE #214: incorporate sorc/fvcom_tools.fd/readme.md into doxygen docs
incorporate sorc/fvcom_tools.fd/readme.md into doxygen docs. This is part of #191

### LINKED ISSUE #219: Remove file sorc/fre-nctools.fd/shared_lib/COPYING
We already have a LICENSE which covers this.

### LINKED ISSUE #268: ENABLE_DOCS option missing from main cmake file?
I expect to see this line, but it's gone:

option(ENABLE_DOCS "Enable generation of doxygen-based documentation." OFF)

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #217 
Part of #191 
Fixes #219
Fixes #214
Fixes #213
Fixes #268

Starting to add list of utilities in user_guide.md. We need a brief description of each of them.

We will use this list to assign testing tasks.

Also some other documentation cleanups.

### CODE REVIEW

**Comment by edwardhartnett** [comment]
@GeorgeGayno-NOAA please give me some descriptions and I will add them...

File: docs/user_guide.md [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
 Utilities for the NCEP models. This is part of the
 [NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS) project.
 
-The UFS_UTILS code here: https://github.com/NOAA-EMC/UFS_UTILS.
+The UFS_UTILS code can be found here:
+https://github.com/NOAA-EMC/UFS_UTILS.
+
+## The Utilities
+
+- exemcsfc_global_sfc_prep.sh
```
Comment (GeorgeGayno-NOAA) [inline docs/user_guide.md:13]: This script runs the emcsfc_ice_blend and emcsfc_snow2mdl programs.  Is it a utility in itself?

**Comment by GeorgeGayno-NOAA** [comment]
> @GeorgeGayno-NOAA please give me some descriptions and I will add them...

* chgres_cube - Creates cold start initial conditions for FV3 model runs
* emcsfc_ice_blend - Blends National Ice Center sea ice cover and EMC sea ice concentration data to create a global sea ice analysis used to update the GFS once per day.
* emcsfc_snow2mdl - Blends National Ice Center snow cover and Air Force snow depth data to create a global depth analysis used to update the GFS snow field once per day.

**Comment by GeorgeGayno-NOAA** [comment]
* fre-nctools - Tools to remap data; and to create the geo-reference fields (latitude, longitude, etc.) for an FV3 grid.
* fvcom_tools - Replaces lake surface and lake ice temperature along with aerial ice concentration generated from the Great Lakes Operational Forecast System (GLOFS) in an FV3 surface restart file.
* global_chgres - Creates cold start initial conditions for FV3 model runs.  Deprecated by the chgres_cube utility.

**Comment by GeorgeGayno-NOAA** [comment]
* global_cycle - Updates the GFS surface conditions using external snow and sea ice analyses.  Updates monthly climatological fields such as plant greenness fraction and albedo.  Runs as part of the GFS and GDAS cycles.
* grid tools - Utilities to filter topography, to create regional extended Schmidt gnomonic grids, and to compute the equivalent global resolution of a regional grid.

File: docs/user_guide.md (hunk 1 above)
Comment (edwardhartnett) [reply docs/user_guide.md:13]: You tell me...

**Comment by GeorgeGayno-NOAA** [comment]
* nst_tf_chg - Initializes the reference temperature used by the NSST model.
* orog_mask_tools - Utilities to create land mask, terrain and gravity wave drag fields; set lake fraction and depth; creates an inland land mask.
* sfc_climo_gen - Creates surface climatological fields, such as vegetation type and albedo, for an FV3 grid.
* vcoord_gen - Generates hybrid coordinate parameters from fields such as surface pressure, model top and the number of vertical levels.  Outputs the 'ak' and 'bk' parameters used by the forecast model to define the hybrid levels.

**Comment by edwardhartnett** [comment]
Thanks, I have added those. There are still: 

- nst_tf_chg

- orog_mask_tools

- sfc_climo_gen

- vcoord_gen

File: docs/user_guide.md (hunk 1 above)
Comment (GeorgeGayno-NOAA) [reply docs/user_guide.md:13]: I would consider that a script and not a utility.

Comment (edwardhartnett) [reply docs/user_guide.md:13]: I removed it from the list.

**Comment by edwardhartnett** [comment]
Let's get these reviewed and merged, and we will move forward further this week...

**Comment by GeorgeGayno-NOAA** [comment]
> Let's get these reviewed and merged, and we will move forward further this week...

@edwardhartnett I don't see these updates in your branch.

**Comment by edwardhartnett** [comment]
OK, I've updated my branch and you can see the changes. Let's get this merged and I will take another pass as I get a better understanding of UFS_UTILS.

File: docs/user_guide.md [hunk 2]
```diff
@@ -1,9 +1,52 @@
...
+  Force snow depth data to create a global depth analysis used to
+  update the GFS snow field once per day.
+
+- fre-nctools - Tools to remap data; and to create the geo-reference
+  fields (latitude, longitude, etc.) for an FV3 grid.
+
+- fvcom_tools - - Replaces lake surface and lake ice temperature along
+  with aerial ice concentration generated from the Great Lakes
+  Operational Forecast System (GLOFS) in an FV3 surface restart
+  file. [fvcom documentation](sorc/fvcom_tools.fd/fvcom_readme.md)
```
Comment (GeorgeGayno-NOAA) [inline docs/user_guide.md:30]: The links to the "fvcom_readme.md" and "global_chgres_user_guide.md" did not work for me.  They are also pointing to your branch.  After the merge, are they automatically updated to point to 'develop'?

File: docs/user_guide.md [hunk 3]
```diff
@@ -1,9 +1,52 @@
...
+- global_cycle - Updates the GFS surface conditions using external
+  snow and sea ice analyses. Updates monthly climatological fields
+  such as plant greenness fraction and albedo. Runs as part of the GFS
+  and GDAS cycles.
+
+- grid_tools - Utilities to filter topography, to create regional
+  extended Schmidt gnomonic grids, and to compute the equivalent
+  global resolution of a regional grid.
+
+- nst_tf_chg
```
Comment (GeorgeGayno-NOAA) [inline docs/user_guide.md:45]: These four utilities have no summary.

Comment (edwardhartnett) [reply docs/user_guide.md:45]: Yes, what should they say?

File: docs/user_guide.md (hunk 2 above)
Comment (edwardhartnett) [reply docs/user_guide.md:30]: Hmmm. Let me check that...

**Comment by GeorgeGayno-NOAA** [comment]
> * nst_tf_chg - Initializes the reference temperature used by the NSST model.
> * orog_mask_tools - Utilities to create land mask, terrain and gravity wave drag fields; set lake fraction and depth; creates an inland land mask.
> * sfc_climo_gen - Creates surface climatological fields, such as vegetation type and albedo, for an FV3 grid.
> * vcoord_gen - Generates hybrid coordinate parameters from fields such as surface pressure, model top and the number of vertical levels.  Outputs the 'ak' and 'bk' parameters used by the forecast model to define the hybrid levels.

@edwardhartnett Don't forget to add these summaries to the user guide.

File: docs/user_guide.md (hunk 3 above)
Comment (edwardhartnett) [reply docs/user_guide.md]: OK, added them and pushed. Thanks!

File: docs/user_guide.md (hunk 2 above)
Comment (edwardhartnett) [reply docs/user_guide.md:30]: OK, fixed it. It doesn't work great in markdown, but the doxygen output is good. This is the same answer Jeff came up with on NCEPLIBS-bufr for dealing with markdown files in the doxygen docs.

**Comment by GeorgeGayno-NOAA** [comment]
> @GeorgeGayno-NOAA I believe this is ready to go and should be merged.

Merge in the latest changes from 'develop' and I will approve.  I assume the doxygen output looks ok?

**Comment by edwardhartnett** [comment]
You should not assume that the doxygen output looks OK. ;-)

Do you have a way to generate and easily examine the documentation? On my system I do -DENABLE_DOCS=On, and then after a build I open the docs locally. Are you on a wcoss machine? Is there even a browser?

I am on a stand-alone linux box...

**Comment by GeorgeGayno-NOAA** [comment]
> You should not assume that the doxygen output looks OK. ;-)
> 
> Do you have a way to generate and easily examine the documentation? On my system I do -DENABLE_DOCS=On, and then after a build I open the docs locally. Are you on a wcoss machine? Is there even a browser?
> 
> I am on a stand-alone linux box...

I have a gh-pages branch.  I will use that.

**Comment by GeorgeGayno-NOAA** [comment]
I built the doxygen: https://georgegayno-noaa.github.io/UFS_UTILS/index.html  It looks ok.  Will merge.

**Review by GeorgeGayno-NOAA — APPROVED** [review]

**Pull request merged**